sdk/python/jobs/automl-standalone-jobs/automl-forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1435 @@
+    "endpoint.defaults.deployment_name = deployment.name\n",
+    "ml_client.batch_endpoints.begin_create_or_update(endpoint).result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 10.3. Invoke the Endpoint\n",
+    "\n",
+    "The next cell contians the command that invokes the endpoint for batch inference job. The `invoke` method contains the `inputs` parameter. This parameter contains the inputs necessary to execute the inference component on the endpoint. To convince yourself this is the case, compare the input parameters for the `inference_component_from_registry` in section 6.3 with the `inputs` we are proving in the next cell. They are identical.\n",
+    "\n",
+    "Notice, the the `forecast_mode` is set to `\"recursive\"`. In the evaluation pipeline this component was used to generate rolling forecast to evalaute model performance on the test set. Here, we are using it to generate a forecast."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = ml_client.batch_endpoints.invoke(\n",
+    "    endpoint_name=endpoint.name,\n",
+    "    deployment_name=deployment.name,\n",
+    "    inputs={\n",
+    "        \"raw_data\": Input(type=AssetTypes.URI_FOLDER, path=\"./data/inference_small\"),\n",
+    "        \"train_experiment_name\": Input(type=\"string\", default=experiment_name),\n",
+    "        \"instance_count\": Input(type=\"integer\", default=1),\n",
+    "        \"max_concurrency_per_instance\": Input(type=\"integer\", default=2),\n",
+    "        \"compute_name\": Input(type=\"string\", default=amlcompute_cluster_name),\n",
+    "        \"forecast_mode\": Input(type=\"string\", default=\"recursive\"),\n",
+    "        \"prs_step_timeout\": Input(type=\"integer\", default=3700),\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we will stream the job output to monitor the execution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_name = job.name\n",
+    "batch_job = ml_client.jobs.get(name=job_name)\n",
+    "print(f\"Batch job status: {batch_job.status}\\n---\")\n",
+    "ml_client.jobs.stream(name=job_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 10.4. Download Forecast Output\n",
+    "\n",
+    "Next, we download the forecast output and print the first few rows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.download(job_name, download_path=\".\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fcst_df = pd.read_csv(output_file, parse_dates=[time_column_name])\n",
+    "fcst_df.head()"
+   ]
+  }
```

Comment:
Let us also add the cell where we delete the endpoint when we are done:
ml_client.online_endpoints.begin_delete(name=endpoint.name).wait()

Reply:
Done.